front/components/Avatar/avatar.spec.tsx

```diff
@@ -0,0 +1,42 @@
+import { render, screen } from '@testing-library/react';
+
+import Avatar from './index';
+
+const mockupUser = {
+  id: '414',
+  username: 'Toto',
+  email: 'toto@gmail.com',
+  description: "I'm an example",
+  packages: [],
+};
+
+test('Avatar T first letter', () => {
+  render(<Avatar user={mockupUser} />);
+
+  const avatar = screen.getByTestId('user-avatar');
```

Comment:
we should avoid using `getByTestId` when not necessary. Is there a better query for this?

Reply:
You are right, I change it by a "getByText("T)" query.

In the query's order priority of testing-library, 'getByText' is the first one that I can use.
(https://testing-library.com/docs/queries/about/#priority)

Reply:
Unfortunately, I don't see an other way :/